modules/gdscript/gdscript.cpp

```diff
@@ -2530,48 +2530,16 @@ void GDScriptLanguage::reload_scripts(const Array &p_scripts, bool p_soft_reload
 
 		if (!p_soft_reload) {
 			//save state and remove script from instances
```

Comment:
While at it, for good measure:
```suggestion
			// Save state and remove script from instances.
```

Reply:
You are right, the method does more then just saving the save. Renaming the method for: `_prepare_script_for_reload`.